src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PremisesEntity.kt

```diff
@@ -171,6 +171,7 @@ class TemporaryAccommodationPremisesEntity(
   @ManyToOne
   @JoinColumn(name = "probation_delivery_unit_id")
   var probationDeliveryUnit: ProbationDeliveryUnitEntity?,
+  var turnaroundWorkingDayCount: Int?
```

Comment:
Since the database column is `NOT NULL` I think this should also be non-nullable

Reply:
Makes sense. I have made changes to make to the field non-nullable